eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,52 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = 'master',
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string] $Dependencies,
+
+    [Parameter()]
+    [string] $TargetPath ="$env:TEMP/$([guid]::NewGuid())",
+
+    [Parameter()]
+    [switch] $DevOpsLogging = ($null -ne $env:SYSTEM_TEAMPROJECTID)
```

Comment:
Can you explain what's the purpose of this and when is this false? What's `env:SYSTEM_TEAMPROJECTID`?

Reply:
I believe in @danieljurek latest changes which he hasn't pushed yet this is removed as we realized it isn't really needed.  However the general idea is that we use this to determine if we are running in DevOps and if so we use different logging for warnings/errors so they show up nicely in Devop logs.